storage/src/cache/filecache/cache_entry.rs

```diff
@@ -1039,14 +1043,25 @@ impl Region {
     }
 }
 
+#[derive(PartialEq)]
+enum FileIoMergeStateStatus {
+    // Last region can take in more io chunks
+    Open,
+    // Last region can no longer take in more io chunks, a new region has to be
+    // created for following chunks.
+    Committed,
+}
+
 struct FileIoMergeState {
     regions: Vec<Region>,
+    status: FileIoMergeStateStatus,
```

Comment:
Can you change `status` to `last_region_joinable: bool` as a marker to tell if the last region in a merge state can join more requests? Also, a few comments to help explain why we need it is even better. Thanks!

Reply:
Sure. It also makes sense.